src/mixins/filing-template-mixin.ts

```diff
@@ -399,8 +408,8 @@ export default class FilingTemplateMixin extends DateMixin {
       } else {
         // Otherwise save default data
         filing.restoration.nameRequest = {
-          legalName: this.getEntitySnapshot.businessInfo.legalName,
-          legalType: this.getEntitySnapshot.businessInfo.legalType
+          legalName: this.getOriginalLegalName,
+          legalType: this.getOriginalLegalType
```

Comment:
Yes, this saves the operating name into the nameRequest object for firms. This is needed for resume a draft correctly.